packages/styles/scss/patterns/blocks/content-group-pictograms/_content-group-pictograms.scss

```diff
@@ -15,6 +15,10 @@
       }
     }
 
+    .#{$prefix}--content-group {
+      margin-bottom: $layout-05;
+    }
+
```

Comment:
hey @RaphaelAmadeu I'm looking at the ticket, and @ltitle also mentioned about pictogram bottom padding needing to be removed for the very last pictogram item

Reply:
@annawen1 these remove the padding from the content-group itself, in the end both would have the same result.

Reply:
I guess that works...I'm a little concerned about other components placed into ContentGroup, will they always have 32px padding bottom?

Reply:
ah I was thinking generically, this is specifically the `_content-group-pictograms` styles so this works! thanks :D